example/app/app.component.scss

```diff
@@ -1,43 +1,5 @@
 @import url('https://fonts.googleapis.com/css?family=Montserrat|Open+Sans');
 
-/* Specific Modal Styles */
```

Comment:
As mentioned above, this is the main reason of the library existing, and can't removed.

The two core principles are:

- Custom CSS. Which doesn't mean no Bootstrap, but rather we aim to be compatible with it.
- Lazy loading content.

Reply:
As mentioned in the import of bootstrap in angular json, what about import bootstrap css in example stylesUrls?